### ISSUE TITLE
[Bug] Lost `Buffer`

### ISSUE THREAD

**Issue by moeshin** [BetterDiscord/BetterDiscord#1676]
### Before Reporting

- [X] I found no existing issues matching my bug
- [X] My issue is not caused by a plugin or theme

### Describe the Bug

brower no have `Buffer`.

### Expected Behavior

no error

### Steps To Reproduce

```js
require('request').get('https://www.gstatic.com/generate_204', console.warn)
```
```text
VM1451 C:\Users\MoeShin\AppData\Roaming\BetterDiscord\data\betterdiscord.asar\preload.js:formatted:224 Uncaught Error: Cannot read properties of undefined (reading 'Buffer')
    at IncomingMessage.<anonymous> (VM1451 C:\Users\MoeShin\AppData\Roaming\BetterDiscord\data\betterdiscord.asar\preload.js:formatted:224:25)
    at IncomingMessage.emit (VM1341 node:events:525:35)
    at endReadableNT (VM1383 readable:1358:12)
    at process.processTicksAndRejections (VM1350 task_queues:83:21)
```

### Operating System

Windows

### Discord Version

Stable

### BetterDiscord Version

v1.9.3

### Anything else?

![image](https://github.com/BetterDiscord/BetterDiscord/assets/28248888/d6507b49-318e-4372-85ae-e56638ed3b1d)

**Comment by ImAFrogOwO** [comment]
Already been discussed.
"Discord removed their Buffer polyfill that we made use of" - Zerebos

**Comment by moeshin** [comment]
fixed bee11905dd1461ac50987177f9885a80e8ceaa55
